src/RawMasterFile.test.cpp

```diff
@@ -145,6 +146,19 @@ TEST_CASE("Parse a master file in .json format", "[.integration]") {
     REQUIRE_FALSE(f.digital_samples());
 }
 
+TEST_CASE("Parse a master file in old .raw format",
```

Comment:
I don't think we need all tags. I would settle on one... maybe swap out to .with-data? 
Selecting tests from one file can be done using -# in the command line enabling file names as tags